src/main/webapp/js/calendar.js

```diff
@@ -17,19 +17,19 @@
 // TODO(#34): Add ability to choose which section the user wants, not just
```

Comment:
was this file meant to be included? It's not mentioned in the PR description.

Reply:
It's the auto formatter. I don't know why it chooses to format files now but not before

Reply:
I can take it out of this PR but some other random PR in the future will get this same problem when they try to auto format.

Reply:
In cases like this, the right approach is to update this in a separate PR (since it does need to be fixed).